File: 프로그래머스/1/12916. 문자열 내 p와 y의 개수/문자열 내 p와 y의 개수.js
function solution(s){
    var answer = true;
    var countp =0;
    var county =0;
    for(let i=0;i<s.length;i++){
        if(s[i].toUpperCase()=="Y") {
            county += 1;
        }else if(s[i].toUpperCase()=="P") {
            countp += 1;
        }
    }
    console.log(countp);
    console.log(county);
    if(county !== countp){
        return false;
    }
    return true;
    
}

File: 프로그래머스/1/12916. 문자열 내 p와 y의 개수/문자열 내 p와 y의 개수.java
class Solution {
    boolean solution(String s) {
        boolean answer = true;
        s = s.toLowerCase();

        int[] count = {0,0};
        
        for(int i=0; i<s.length(); i++){
            if(s.charAt(i) == 'p'){
                count[0]++;
            }else if(s.charAt(i) == 'y'){
                count[1]++;
            }
        }
        
        answer = count[0]==count[1];
        
        return answer;
    }
}